dev/tasks/conda-recipes/pyarrow/meta.yaml

```diff
@@ -44,6 +44,8 @@ requirements:
 test:
   imports:
     - pyarrow
+    - pyarrow.fs
+    - pyarrow.s3fs
```

Comment:
Aside: we worried about the naming conflict with https://s3fs.readthedocs.io/en/latest/? Might be good to disambiguate